### PR TITLE
Solve issue #194 .

### PR DESCRIPTION
**Pull request by qpakzk**
Hello @namhyung , I send this PR.
I solved issue #194 .
First, I added error handling for return values from fopen  in session.c.
Second, I changed fuctions from to pr_out in symbol.c.

Signed-off-by: Sangwon Hong <qpakzk@gmail.com>

### CODE REVIEW

File: utils/session.c [hunk 1]
```diff
@@ -911,10 +911,16 @@ TEST_CASE(task_symbol)
 	FILE *fp;
 
 	fp = fopen("sid-test.map", "w");
+    if(fp == NULL)
+        pr_err("cannot open %s file", "sid-test.map");
```
Comment (namhyung) [inline utils/session.c]: Please check your editor settings - we use TAB size 8 for indentation.

Also you'd better using `TEST_NE(fp, NULL);` instead.

**Comment by qpakzk** [comment]
I edited to TEST_NE(fp, NULL) and indentation the way you had pointed out.

**Comment by namhyung** [comment]
LGTM

**Comment by elfring** [comment]
There are still more source code places to reconsider, aren't there?

**Pull request merged**